README.md

```diff
@@ -28,21 +28,20 @@
 
 Anyone and everyone is welcome to contribute and help out with the project!
 
-To contribute, please follow the [contributing guide](https://github.com/Paneedah/Modern-Warfare-Cubed/blob/master/CONTRIBUTING.md).
+To contribute, please read the [contributing guide](https://github.com/Paneedah/Modern-Warfare-Cubed/blob/master/CONTRIBUTING.md).
```

Comment:
Should stay as follow as the guide explicitly put guideline to follow like making a new branch for each changes, *which apparently you did not follow*